workspace_tools/export/codered_lpc4088_cproject.tmpl

```diff
@@ -981,7 +981,7 @@
                             <tool id="com.crt.advproject.gas.exe.release.579950187" name="MCU Assembler" superClass="com.crt.advproject.gas.exe.release">
                                 <option id="com.crt.advproject.gas.arch.1401271875" name="Architecture" superClass="com.crt.advproject.gas.arch" value="com.crt.advproject.gas.target.cm3" valueType="enumerated"/>
                                 <option id="com.crt.advproject.gas.thumb.1024544278" name="Thumb mode" superClass="com.crt.advproject.gas.thumb" value="true" valueType="boolean"/>
-                                <option id="gnu.both.asm.option.flags.crt.637466836" name="Assembler flags" superClass="gnu.both.asm.option.flags.crt" value="-c -x assembler-with-cpp -D__NEWLIB__  -DNDEBUG -D__CODE_RED " valueType="string"/>
+                                <option id="gnu.both.asm.option.flags.crt.637466836" name="Assembler flags" superClass="gnu.both.asm.option.flags.crt" value="-c -x assembler-with-cpp -D__NEWLIB__  -DNDEBUG -D__CODE_RED -DFPU_PRESENT=1 " valueType="string"/>
```

Comment:
I will test it tomorrow at work. However I'm not sure that it will work. The LPCXpresso IDE has completely different ways of specifying flags/symbols for asm (i.e. the GUI has lists of symbols for c/c++ but for assembler it is only one input field). I'll post my findings when I have tested it.